pkg/process/config/config_windows.go

```diff
@@ -13,7 +13,7 @@ var (
 	defaultLogFilePath = "c:\\programdata\\datadog\\logs\\process-agent.log"
 
 	// Agent 6
-	defaultDDAgentBin = "c:\\Program Files\\Datadog\\Datadog Agent\\embedded\\agent.exe"
+	defaultDDAgentBin = "c:\\Program Files\\Datadog\\Datadog Agent\\bin\\agent.exe"
```

Comment:
When did this location change? Is there a PR? I wonder if it broke a previous build?

Reply:
@derekwbrown might know better on this